### DISCUSSION TITLE
'DDRG' was not declared in this scope updating HW3 to V9.4

### DISCUSSION THREAD

**Discussion by hitmanmcc** [sanni/cartreader#461] [Need help]
Below is a screenshot and a partial copy of the error messages (too long for GitHub). Feel free to reach out for additional information I can help with.

![image](https://user-images.githubusercontent.com/3464842/183643213-c4c04d35-0467-49d7-b5d6-0615424a6eb2.png)

C:\Users\andre\OneDrive\Ambiente de Trabalho\V9.4_Portable\Arduino IDE Portable\portable\sketchbook\Cart_Reader\WS.ino: In function 'void dataOut_WS()':

WS:1321:3: error: 'DDRA' was not declared in this scope

   DDRA = 0xff;

   ^~~~

C:\Users\andre\OneDrive\Ambiente de Trabalho\V9.4_Portable\Arduino IDE Portable\portable\sketchbook\Cart_Reader\WS.ino:1321:3: note: suggested alternative: 'DDRB'

   DDRA = 0xff;

   ^~~~

   DDRB

exit status 1

'DDRG' was not declared in this scope

This report would have more information with
"Show verbose output during compilation"
option enabled in File -> Preferences.

**Comment by sanni** [comment]
Please check if you have selected Arduino Mega under Tools -> Board:

![image](https://dl.dropboxusercontent.com/s/3az5gr44boswylf/megaselect.jpg?dl=1)

**Comment by hitmanmcc** [comment]
That was it, thanks!
